loopdb/store.go

```diff
@@ -71,6 +81,8 @@ var (
 	byteOrder = binary.BigEndian
 
 	keyLength = 33
+
+	defaultProtocolVersion = looprpc.ProtocolVersion_USER_EXPIRY_LOOP_OUT
```

Comment:
So if protocol version is not written for a swap we use the latest version, couldn't that report a legacy client which isn't on this version yet as supporting it?

Reply:
For all we know this is undecidable, but also because we don't actually serialize or send the version for swaps that don't have it serialized, it's just a matter of preference. We can safely change it to legacy as well, as it really only is used to decide between the two script versions.

Reply:
Decided to instead go with the enum translation and add the the unrecorded similarly as on the server side. PTAL